src/components/result-error/result-error.component.ts

```diff
@@ -0,0 +1,10 @@
+
+export const resultError: angular.IComponentOptions = {
+  bindings: {
+    error: '=',
+    actionType: '@',
+    serviceName: '=',
```

Comment:
Is this component only for use with service errors? Could this be any object's name?